packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.styles.ts

```diff
@@ -60,3 +60,19 @@ export const StyledDocsLinkContainer = styled(Box, {
     marginTop: theme.spacing(2),
   },
 }));
+
+export const StyledTiersDocsLinkContainer = styled(StyledDocsLinkContainer, {
+  label: 'StyledTiersDocsLinkContainer',
+})(({ theme }) => ({
+  [theme.breakpoints.down(995)]: {
+    marginLeft: 'unset',
+    marginTop: theme.spacing(2),
+  },
+}));
```

Comment:
Can we just add something like `justifyContent="space-between"` to the parent Stack instead of adding marginLeft at a specific breakpoint?

```suggestion
export const StyledTiersDocsLinkContainer = styled(Box, {
  label: 'StyledTiersDocsLinkContainer',
})(({ theme }) => ({
  [theme.breakpoints.down('md')]: {
    marginTop: theme.spacing(2),
  },
}));
```